### DISCUSSION TITLE
Spring for GraphQL vs graphql-java-kickstart

### DISCUSSION THREAD

**Discussion by CAWilson94** [graphql-java/graphql-java#3442] [General]
Hi there, we are on the older kickstart version of spring graphql. Has anyone noticed a difference when changing over to the newer Spring for GraphQl library? In particular, I am trying to find a good way to decrease the latency of one of our queries. In all fairness we are asking it for 100k accounts, with a schema type of > 10 properties without nesting.

**Comment by dondonz** [comment]
I'm not sure about the performance differences, but I can say Spring for GraphQL is the official Spring integration and it's actively maintained. It was built with the GraphQL Java team so that's our recommendation if you're using Spring.

I have few details about your use case but sending fewer bytes down the wire is always preferable, if you can implement pagination to reduce the total response size, that may help you.
